test/string.carp

```diff
@@ -323,4 +323,12 @@
                 &[\� \�]
                 &(chars &(from-bytes &[255b 255b]))
                 "check for invalid UTF-8 sequences")
+  (assert-equal test
+                "hällo wörld"
+                &(ascii-to-lower "HälLo WöRld")
+                "ascii-to-lower works for valid UTF-8" )
```

Comment:
I find the test case here a bit confusing. The function purportedly only works on ASCII values, but we test the function against extended ASCII values here and mention UTF8 values in this test?

Reply:
Well hopefully not _that_ confusing:
The sources are encoded in UTF-8, so only the ASCII-compatible bits are affected by the transformation, the rest is not changed. It's really just the underlying C library at work (with all its quirks).

Reply:
Ah, that makes sense, just threw me for a loop on a first skim. Thanks!